src/utils/path.ts

```diff
@@ -96,13 +96,15 @@ export async function findFiles(
   const paths = Array.isArray(fileNames) ? fileNames : [fileNames]
   const localFS = fs.cwd(cwd)
 
-  return localFS.findAsync({
+  const files = await localFS.findAsync({
     matching: [
       ...paths,
       ...baseIgnores,
       ...(config?.ignore?.map(i => `!${i}`) ?? []),
     ],
   })
+
+  return files.map(f => (path.isAbsolute(f) ? f : localFS.path(f)))
```

Comment:
Does this need a code comment? Seems that not all results of `findAsync` are absolutely paths? Is that weird?

Reply:
Well, I think it depends on the file pattern passed. In our case, because we pass `./` the results are also relative. This makes sure paths are always absolute. Will add a comment.